ui/job-view/pushes/Push.jsx

```diff
@@ -309,13 +310,32 @@ class Push extends React.PureComponent {
       // remove old versions of jobs we just fetched.
       const existingJobs = jobList.filter((job) => !newIds.includes(job.id));
       // Join both lists and add test_paths and task_run property
-      const newJobList = [...existingJobs, ...jobs].map((job) => {
+      const joinedJobList = [...existingJobs, ...jobs].map((job) => {
         if (Object.keys(manifestsByTask).length > 0) {
           job.test_paths = manifestsByTask[job.job_type_name] || [];
         }
         job.task_run = getTaskRunStr(job);
         return job;
       });
+      // If the pageload job has a side-by-side comparison associated
+      // add job.hasSideBySide containing sideBySideJob.job_type_name
+      const newJobList = joinedJobList.map((job) => {
```

Comment:
```suggestion
      // eslint-disable-next-line camelcase
      const newJobList = joinedJobList.map((job) => {
```

Adding the disable before, like suggested, will apply to all the code bellow meaning there's no need to repeat the line every time `job_type_name` is used. Another option is to move it at the top of the file, then it will apply to the entire file.

Reply:
javascript-tests don't agree with this.

Reply:
![image](https://user-images.githubusercontent.com/47977085/228221293-6293cc00-8305-4578-a47a-41c76e41ed87.png)

Reply:
Could you try `/* eslint-disable @typescript-eslint/camelcase */ ` at the top of the file?